cockroachcloud/frequently-asked-questions.md

```diff
@@ -18,6 +18,15 @@ The allowlist is comprised of IP addresses that you provide to us, and is an add
 
 We use separate certificate authorities for each cluster, and all connections to the cluster over the internet use TLS 1.2.
 
+## Is encryption-at-rest enabled on CockroachCloud?
+
+Yes. All data on CockroachCloud is encrypted-at-rest using the tools provided by the cloud provider that your cluster is running in.
+
+- For GCP, we use [persistent disk encryption](https://cloud.google.com/compute/docs/disks#pd_encryption).
+- For AWS, we use [EBS encryption-at-rest](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/EBSEncryption.html).
+
+However, we do not enable encryption-at-rest within CockroachDB by default, so encryption will appear to be disabled in the [DB Console](../stable/ui-overview.html) since it is unaware of cloud provider encryption.
```

Comment:
This would read a bit more clearly to me as something like "we do not enable CockroachDB's internal implementation of encryption at rest (link?) since we are relying on the cloud provider's implementation (as noted above).  This means that encryption will appear to be disabled ..."

IDK if there is any CRDB/CC doc to link for our own encryption at rest implementation (hence the '(link?)' fragment above)